--- Makefile.am_orig	2025-11-14 14:56:45.165378687 +0200
+++ Makefile.am	2025-11-14 15:00:26.719264563 +0200
@@ -20,7 +20,7 @@
 
 ## system requires a shared libconfig
 AM_CFLAGS = $(LIBCONFIG_CFLAGS) $(LIBNL_CFLAGS)
-AM_LDFLAGS = $(LIBCONFIG_LIBS) $(LIBNL_LIBS) -lrt
+AM_LDFLAGS = $(LIBCONFIG_LIBS) $(LIBNL_LIBS) -lrt -static
 
 ## header files to be installed, for programs using the client interface to lldpad 
 lldpad_includedir= ${includedir}/lldpad
@@ -86,14 +86,14 @@
 
 vdptool_SOURCES = vdptool.c lldp_util.c qbg/vdp22_clif.c vdptool_cisco_oui.c
 vdptool_LDADD = liblldp_clif.la
-vdptool_LDFLAGS = -llldp_clif $(LIBNL_LIBS)
+vdptool_LDFLAGS = -llldp_clif $(LIBNL_LIBS) -static
 
 dcbtool_SOURCES = dcbtool.c dcbtool_cmds.c parse_cli.l \
 	weak_readline.c lldp_rtnl.c lldp_util.c \
 	$(lldpad_include_HEADERS) $(noinst_HEADERS)
 
 dcbtool_LDADD = liblldp_clif.la
-dcbtool_LDFLAGS = -ldl -llldp_clif $(LIBNL_LIBS)
+dcbtool_LDFLAGS = -ldl -llldp_clif $(LIBNL_LIBS) -static
 
 lldptool_SOURCES = lldptool.c lldptool_cmds.c lldp_rtnl.c \
 		   lldp_mand_clif.c lldp_basman_clif.c lldp_med_clif.c \
@@ -102,7 +102,7 @@
 		   lldp_orgspec_clif.c lldp_cisco_clif.c lldp_evb22_clif.c \
 		   weak_readline.c $(lldpad_include_HEADERS) $(noinst_HEADERS)
 lldptool_LDADD = liblldp_clif.la
-lldptool_LDFLAGS = -ldl -llldp_clif $(LIBNL_LIBS)
+lldptool_LDFLAGS = -ldl -llldp_clif $(LIBNL_LIBS) -static
 
 if BUILD_DEBUG
 nltest_SOURCES = test/nltest.c test/nltest.h
